capstone-frontend/src/components/CreateList.jsx

```diff
@@ -1,23 +1,23 @@
 import React, { Component } from 'react';
-import { Button, Form, Spinner } from 'react-bootstrap'
+import { withRouter } from 'react-router-dom'
+import { Button, Form, Spinner, Modal } from 'react-bootstrap'
 
 class CreateList extends Component {
 
   constructor(props) {
     super(props)
 
     this.state = {
-      loading: false
+      loading: false,
+      showModal: false
     }
   }
 
-  handleSubmit = async (event) => {
+  handleSubmit = async () => {
 
     this.setState({ loading: true })
 
-    event.preventDefault();
```

Comment:
Do you not need to prevent the default form submit behavior? Omitting this generally triggers a page reload.

Reply:
For the modal form, I am using ```onClick``` for submitting, thus I can't use ```event.preventDefault```. When I was still using ```onSubmit``` I did needed ```event.preventDefault()```. Unfortunately, ```onSubmit``` generates issues when submitting on a Modal, for example, depending on the page, it may or may not actually submit the form.